mkctr.go

```diff
@@ -96,6 +98,7 @@ func main() {
 		ldflagsArg  = flag.String("ldflags", "", "the --ldflags value to pass to go")
 		gotags      = flag.String("gotags", "", "the --tags value to pass to go")
 		push        = flag.Bool("push", false, "publish the image")
+		outPath     = flag.String("out-path", "", "write single images to a tarball, or multi-platform images to a folder")
```

Comment:
Do you need the single file path option? Can't we have a single `--dst` flag that takes a directory arg and writes either one or many files to that directory depending on whether it's a multi-arch or not?

Reply:
Feels like that would simplify the UX

Reply:
Fixed